libcudacxx/include/cuda/std/__internal/features.h

```diff
@@ -24,6 +24,8 @@
 #define _LIBCUDACXX_HAS_EXTERNAL_ATOMIC_IMP()   1
 #define _LIBCUDACXX_HAS_MONOTONIC_CLOCK()       0
 #define _LIBCUDACXX_HAS_SPACESHIP_OPERATOR()    0
+#define _LIBCUDACXX_HAS_MULTIARG_OPERATOR_BRACKETS() \
```

Comment:
I believe this should actually be more of an `_CCCL_HAS_MULTIARG_OPERATOR_BRACKETS`

There are no libcu++ specific constraints here